model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -58,13 +62,23 @@ sealed class SpdxExpression {
             return visitor.visit(parser.licenseExpression())
         }
     }
+
+    /**
+     * Return all valid SPDX licenses contained in this expression. LicenseRefs and invalid licenses are ignored.
+     *
+     * TODO: The result should also contain SPDX license exceptions associated to the licenses.
```

Comment:
Why not add the TODO comment to the class instead of to this function?
This would leave the freedom whether a separate function for license exceptions is introduced vs extending the result data structure.

From the TODO comment it may be a bit unclear `why` the result should contain the exceptions.

Reply:
I have removed the TODO instead, there will be many changes to this class in the near future anyway, which will also address this issue.